src/hash.h

```diff
@@ -112,23 +112,38 @@ class CHashWriter {
     int GetVersion() const { return nVersion; }
 
     void write(const char *pch, size_t size) {
-        ctx.Write({(const uint8_t *)pch, size});
+        ctx.Write((const uint8_t *)pch, size);
```

Comment:
Why does this need to change? Seems like an optimization of some sort?

Reply:
No, they just have different signatures:

```
CHash256 &Write(Span<const uint8_t> input);
vs
CSHA256 &Write(const uint8_t *data, size_t len);
```

🤷‍♂️